### PR TITLE
Add Draft.js

### PR DESCRIPTION
**Pull request by nikgraf**
I tried to add it early March when the list was too young. I have maintained the list since then and would be happy to add it here.

This was the first PR https://github.com/sindresorhus/awesome/pull/564

### CODE REVIEW

**Comment by pokeball99** [comment]
@nikgraf he would usually just reopen the first issue for cases like yours, no real need to open a new PR

**Comment by sindresorhus** [comment]
@pokeball99 Sure, wanna do a pull request?

**Comment by sindresorhus** [comment]
- Be consistent about the naming. You're using all of these: DraftJS, Draft-JS, Draft JS, Draft.js, DRAFTJS. `Draft.js` is the correct format.
- Add a succinct intro at the top describing what DraftJS is and linkify the name to its website.
- Don't use `---` between the sections. It's already clear enough with the headings.
- Add a description to each item.
- You should include a link to their Slack channel: https://draftjs.herokuapp.com/

File: readme.md [hunk 1]
```diff
@@ -177,6 +177,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Ionic Framework 2](https://github.com/candelibas/awesome-ionic2)
 - [Chrome DevTools](https://github.com/ChromeDevTools/awesome-chrome-devtools)
 - [PostCSS](https://github.com/jjaderg/awesome-postcss)
+- [DraftJS](https://github.com/nikgraf/awesome-draft-js)
```
Comment (sindresorhus) [inline readme.md]: Add a description after. See other items with description for example on how to do it.

Comment (sindresorhus) [inline readme.md]: Draft.js

**Comment by nikgraf** [comment]
@sindresorhus just executed on all of the suggestions, makes the whole list nicer, thx 🙌

**Comment by hackerkid** [comment]
@nikgraf You should probably squash those 2 commits into one.

**Comment by nikgraf** [comment]
@hackerkid just did so …

**Comment by sindresorhus** [comment]
Landed, but still some minor things that needs fixing:

- You need to update the naming in the repo description too.
- The `Resources` link in the table of contents doesn't go anywhere.
- `Draft.js Slack channel` => `Slack channel`. Draft.js is already implied.
- Be consistent about capitalization of the header. It's currently very inconsistent. Use http://titlecapitalization.com/ to get it right.
- `A Beginner’s Guide to Draft JS` => `A Beginner’s Guide to Draft.js`
- All item descriptions should begin with a uppercase character and en in a dot. E.g. `- Description of item.`

**Comment by nikgraf** [comment]
@sindresorhus thx, just fixed all the points you mentioned - thx for taking the time 🙌

**Pull request closed without merging**